src/reuse/helper/errorMessages.ts

```diff
@@ -12,7 +12,7 @@ export class ErrorMessages {
   }
 
   public static customErrorWithoutMessage(functionName: string): string {
-    return `Function '${functionName}' failed with unknown error`;
+    return `Function '${functionName}' failed with : unknown error`;
```

Comment:
Please remove the spaces before the ":" of the error messages.
```suggestion
    return `Function '${functionName}' failed with: unknown error`;
```

Reply:
done